sv_pipeline/genome/load_data.py

```diff
@@ -1,277 +0,0 @@
-# A simple usage guide:
```

Comment:
I might be being aggressive/presumptuous with removing these, but I don't think we'll be going backwards at this point?